### ISSUE TITLE
Add Hugo version to README

### ISSUE THREAD

**Issue by nobeeakon** [OSGeo/grass-website#312]
Based on  [this comment](https://github.com/OSGeo/grass-website/pull/310#issuecomment-1201715419) I guess it would be between v0.69.0 and < [v0.100.2](https://www.opensourceagenda.com/projects/hugo/versions) as v0.100.2 is not using the current defaultMarkdownHandler

**Comment by neteler** [comment]
On grass.osgeo.org, we have currently (manually) installed:

```
hugo version
Hugo Static Site Generator v0.69.0-4205844B linux/amd64 BuildDate: 2020-04-10T09:12:34Z

# OS: Debian GNU/Linux 10 (buster)
```

If it doesn't break anything I can update it to a more recent version.

**Comment by veroandreo** [comment]
I've been testing locally with 0.83 and haven't seen anything broken... so up to that version might be safe :) @nobeeakon have you tested with newer versions?

**Comment by echoix** [comment]
I've bumped into the problem of having the blackfriday markdown renderer specified in the configuration when trying to setup a GitPod/GitHub Codespaces to easily contribute to the website. 
Hugo's current version is now at v0.110. Since v0.100.2, the long deprecated blackfriday renderer was completely removed. (June 8, 2022). The default and Commonmark-Compliant renderer is Goldmark since v0.61 (https://github.com/gohugoio/hugo/issues/5963 in 2019). 

What are the differences in the build output when using one or the other? (Like a diff of the output of what the server deploys vs what we can build locally). Appart from the handling of invalid names for the `#` part of the links, what does it do?

We are now running with an already unsupported configuration, it's only working since the versions used are older.

Why does the hosting server need to have a Hugo version? Doesn't a deploy mean to transfer the build contents to a web server, and not build it there where the access to debug what's going on is more limited?

**Comment by neteler** [comment]
> Why does the hosting server need to have a Hugo version? Doesn't a deploy mean to transfer the build contents to a web server, and not build it there where the access to debug what's going on is more limited?

We do not (yet?) build the hugo pages here on Github.

So far this repo is checked out on grass.osgeo.org, then hugo is run there and the resulting pages moved into the Apache http directory.

Suggestions to do all this in a better way are most welcome.

**Comment by veroandreo** [comment]
> I've bumped into the problem of having the blackfriday markdown renderer specified in the configuration when trying to setup a GitPod/GitHub Codespaces to easily contribute to the website. Hugo's current version is now at v0.110. Since v0.100.2, the long deprecated blackfriday renderer was completely removed. (June 8, 2022). The default and Commonmark-Compliant renderer is Goldmark since v0.61 ([gohugoio/hugo#5963](https://github.com/gohugoio/hugo/issues/5963) in 2019).
> 
> What are the differences in the build output when using one or the other? (Like a diff of the output of what the server deploys vs what we can build locally). Appart from the handling of invalid names for the `#` part of the links, what does it do?

I edited the config file as in #310, i.e., to use goldmark and at least locally I do not see issues in the build with hugo 0.83. What problems do you bump into? What is changed with newer hugo versions if you switch to goldmark?

> We are now running with an already unsupported configuration, it's only working since the versions used are older.

What would be the benefits of an update to newer hugo versions? Would you be willing to lead such an effort?

> Why does the hosting server need to have a Hugo version? Doesn't a deploy mean to transfer the build contents to a web server, and not build it there where the access to debug what's going on is more limited?

Do you have experience with other settings or suggestions on how to do it more efficiently?

**Comment by neteler** [comment]
Update: we are planning to update grass.osgeo.org to Debian bookworm which ships hugo 0.111.3 according to
https://repology.org/badge/vertical-allrepos/hugo-sitegen.svg

Might the upgrade cause any troubles here?

**Comment by veroandreo** [comment]
I have hugo v0.98.0 and it builds alright, must test with a newer version. @nobeeakon have you tried with the version mentioned by @neteler or newer?

**Comment by echoix** [comment]
> Hugo's current version is now at v0.110. Since v0.100.2, the long deprecated blackfriday renderer was completely removed. (June 8, 2022). The default and Commonmark-Compliant renderer is Goldmark since v0.61 (gohugoio/hugo#5963 in 2019).

The breaking change is with v100.2

**Comment by veroandreo** [comment]
> > Hugo's current version is now at v0.110. Since v0.100.2, the long deprecated blackfriday renderer was completely removed. (June 8, 2022). The default and Commonmark-Compliant renderer is Goldmark since v0.61 ([gohugoio/hugo#5963](https://github.com/gohugoio/hugo/issues/5963) in 2019).
> 
> The breaking change is with v100.2

Thanks @echoix, is it clear for you what we need to do then? Would you be willing to test and create a PR? We can then merge once the update is in place?

**Comment by echoix** [comment]
@veroandreo I'm on the road this week, so I can only do things on a phone, like searches and al.

If I was on a computer, I'd build the website with the current Hugo version, copy the whole output contents in another folder, then redo the same thing with the current version + switching the markdown renderer, copy the build outputs, and compare the two outputs with something like `git diff`. I'd give a little more attention to the URLs generated containing `#` (anchors), as that was at the time a place that there could be a different behaviour in special cases, thus check if we are using these special cases.

**Comment by petrasovaa** [comment]
Didn't see this issue before I created #394. I had issues with raw HTML not being rendered, so e.g. images were not shown at all, it needs to be allowed in the configuration. The rest looked fine, but I didn't check everything.

**Comment by neteler** [comment]
Closed by #394
